ci/params/aurora/taskcat.yml

```diff
@@ -2,12 +2,12 @@
 global:
```

Comment:
Because there is no Aurora option, we don't need this parameter set for CI :)

Reply:
Nice spot, removed.